src/details/ArborX_Ray.hpp

```diff
@@ -72,8 +115,8 @@ struct Ray
   KOKKOS_FUNCTION
   constexpr Vector const &direction() const { return _direction; }
 
-  Point _origin = {};
-  Vector _direction = {{0.f, 0.f, 0.f}};
+  // FIXME avoid breaking Wenjun's code
+  using Vector [[deprecated]] = Vector;
```

Comment:
@wjge make sure you see this

Reply:
```suggestion
  // FIXME avoid breaking Wenjun's code
  using Vector [[deprecated]] = ArborX::Experimental::Vector;
```
to avoid confusions.

Reply:
What confusion?

Reply:
I find it slightly confusing to read `Vector = Vector`.